assets/js/modules/tagmanager/components/common/AccountSelect.js

```diff
@@ -61,13 +61,15 @@ export default function AccountSelect() {
 		>
 			{ ( accounts || [] )
 				.concat( {
+					/* eslint-disable sitekit/camelcase-acronyms */
 					accountId: ACCOUNT_CREATE,
 					name: __( 'Set up a new account', 'google-site-kit' ),
 				} )
 				.map( ( { accountId, name } ) => (
 					<Option
 						key={ accountId }
 						value={ accountId }
+						/* eslint-enable sitekit/camelcase-acronyms */
```

Comment:
See above. While here there is no actual instance where this rule should be enforced _right now_, I still think multi-line exclusions are prone to future errors, so I think here as well we should annotate individual lines.